estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+from dateutil.relativedelta import relativedelta
+from odoo import models , fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "This is property tabel."
+
+    name = fields.Char("Property Name", required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date(string="Available From", copy=False, default=fields.Date.today()+relativedelta(months=3))
```

Comment:
```suggestion
    date_availability = fields.Date(string="Available From", copy=False, default=fields.Date.today()+relativedelta(months=3))
```
1. what is the purpose of `copy` attribute
2. can we use lambda function here.

Reply:
1. The copy attribute in Odoo is used to control whether a field's value is copied when duplicating a record.
2. Yes we can use lambda function here.